docs/diagrams/InsertClassDiagram.puml

```diff
@@ -80,8 +51,6 @@ class InsertCommand implements Model {
 }
```

Comment:
Line 42 - 45 I think is advisable to delete all these static string as mentioned in #308 where he mentioned `Remove getter/setter functions, trivial data members`
I think the rest of us didn't include the static string messages.